pages/code-review.md

```diff
@@ -114,12 +149,26 @@ When reviewing code, you should be able to check off each of the following:
     - [ ] Do the changes address the project's needs?
     - [ ] Do the changes respect the project's existing style?
     - [ ] Does the new code avoid reproducing existing functionality?
-    - [ ] Are functions as simple as possible?
+    - [ ] Are functions/classes as simple as possible?
     - [ ] Is the code as efficient as possible?
-    - [ ] Is the usage of each function clear?
+    - [ ] Is the usage of each function/class clear?
     - [ ] Have edge cases been considered and tested for?
-
-Use this bookmarklet code to add this checklist to your Pull Request comments: [Code Review Checklist](https://gist.github.com/cfarm/a4174fe6f775353a3115)
+    - [ ] Does the code represent a logical unit of work?
+    - [ ] Are there any glaring syntax errors that were missed?
+    - [ ] Are language constructs being utilized properly?
+    - [ ] Are there no unecessary premature optimizations?
+    - [ ] Are any frameworks/libraries being used leveraged properly?
+    - [ ] Are the changes clean and would they pass through a linter?
+    - [ ] Are there useful comments/documentation where needed?
+    - [ ] Is the commit message well-formed with succinct details about
+          what the change is intended for?
+
+In adition to this checklist, you should also pull down the actual changes
+and run them manually to put them through their paces.  Does the code
+actually do what you expect it to do after reading through it?
+
+Use this bookmarklet code to add this checklist to your Pull Request comments:
+[Code Review Checklist](https://gist.github.com/cfarm/a4174fe6f775353a3115)
```

Comment:
This is redundant with the earlier link, hey? Also, do you think we should link there at all, if we now have a different checklist?